Maple2.Server.Game/Manager/Items/EquipManager.cs

```diff
@@ -193,6 +203,26 @@ public class EquipManager {
         }
     }
 
+    /// <summary>
+    /// Unequips an cosmetic item (Hair, Eyes, Face Decor) and discards the previous cosmetic item.
+    /// </summary>
+    public bool EquipCosmetic(Item cosmetic, EquipSlot equipSlot) {
+        lock (session.Item) {
+            Item? currentCosmetic = session.Item.Equips.Get(equipSlot);
```

Comment:
Here and below `session.Item.Equips.*` is redudant, you are already in the class so you can all the function directly.